src/assets/scss/_form.scss

```diff
@@ -79,3 +79,14 @@
     color: $black;
   }
 }
+
+.dropdown-group-wrapper {
+  position: relative;
+  z-index: 2;
```

Comment:
can we try to use some of the `z-index` that [Paragon](https://github.com/openedx/paragon/blob/master/scss/core/_variables.scss#L792) provides?